hoomd/hpmc/ShapeFacetedEllipsoid.h

```diff
@@ -284,14 +285,16 @@ struct ShapeFacetedEllipsoid
         vec3<OverlapReal> e_y(0,1,0);
         vec3<OverlapReal> e_z(0,0,1);
         quat<OverlapReal> q(orientation);
-        vec3<OverlapReal> s_x = rotate(q, sfunc(rotate(conj(q),e_x))+vec3<OverlapReal>(params.origin));
-        vec3<OverlapReal> s_y = rotate(q, sfunc(rotate(conj(q),e_y))+vec3<OverlapReal>(params.origin));
-        vec3<OverlapReal> s_z = rotate(q, sfunc(rotate(conj(q),e_z))+vec3<OverlapReal>(params.origin));
+        vec3<OverlapReal> s_x_plus = rotate(q, sfunc(rotate(conj(q),e_x))+vec3<OverlapReal>(params.origin));
+        vec3<OverlapReal> s_y_plus = rotate(q, sfunc(rotate(conj(q),e_y))+vec3<OverlapReal>(params.origin));
+        vec3<OverlapReal> s_z_plus = rotate(q, sfunc(rotate(conj(q),e_z))+vec3<OverlapReal>(params.origin));
+        vec3<OverlapReal> s_x_minus = rotate(q, sfunc(rotate(conj(q),-e_x))+vec3<OverlapReal>(params.origin));
+        vec3<OverlapReal> s_y_minus = rotate(q, sfunc(rotate(conj(q),-e_y))+vec3<OverlapReal>(params.origin));
+        vec3<OverlapReal> s_z_minus = rotate(q, sfunc(rotate(conj(q),-e_z))+vec3<OverlapReal>(params.origin));
```

Comment:
Can ellipsoids be asymmetric? I don't understand the reason for this change.

Reply:
Never mind, the issue arises for ellipsoid unions. That makes sense, I retract the comment.